Admin/BlockAdmin.php

```diff
@@ -243,7 +241,14 @@ public function getPersistentParameters()
      */
     public function toString($object)
     {
-        return $object->getName();
+        if (!is_object($object)) {
+            return '';
```

Comment:
> This is consistent with implementation of toString in parent Admin class of SonataAdmin ...

Well that is sad news… if the variable can actually not be an object, it should not be named `$object` and there should not be any exception IMO. And maybe a cast to string can be better

> What about doing this in another PR on master @greg0ire ?

Well there is no BC-break here, so if we can do it clean from the start. Is there a valid reason to copy the implementation of `Admin::toString` BTW ?

To answer all this, I would need an explanation of the use case for calling this method with something else than an object.